docs/install/quick-start-guide.md

```diff
@@ -0,0 +1,148 @@
+---
+title: "Quick Start Guide - Non-Production"
+linkTitle: "Quick Start Guide"
+weight: 1
+---
+
+This guide will provide step by step instructions on how to install Ondat onto your cluster, with our helm chart, for a non-production environment.
+
+> ⚠️ This guide is for a non-production installation. Please follow our [other installation guides](https://docs.ondat.io/docs/install/) for a production-ready installation of Ondat
+
+## Prerequisites
+
+* [kubectl](https://kubernetes.io/docs/tasks/tools/#kubectl)
+* [helm](https://helm.sh/docs/intro/install/)
+
+This guide assumes you already have a Kubernetes cluster, with **at least** 3 worker nodes.
+
+### This guide has been tested on the following Kubernetes distributions
+
+* Vanilla Kubernetes
+* Rancher
+
+## Step 1 - Install Ondat Helm Charts
+
+Add the Ondat chart repository to Helm:
+
+```bash
+helm repo add ondat https://ondat.github.io/charts
+helm repo update
+```
+
+## Step 2 - Install Local Path Provisioner
+
+Etcd requires a storage class before we can start Ondat. For non-production environments the Local Path Provisioner storage class can be used.
+
+```bash
+kubectl apply --filename="https://raw.githubusercontent.com/rancher/local-path-provisioner/v0.0.21/deploy/local-path-storage.yaml"
+```
+
+The following command can be used to ensure the Local Path Provisioner was successfully deployed. The expected output is also shown
+
+```bash
+> kubectl get pod,storageclass --namespace=local-path-storage
+NAME                                         READY   STATUS    RESTARTS   AGE
+pod/local-path-provisioner-c4d687f4c-bxmjt   1/1     Running   0          3h10m
+
+NAME                                     PROVISIONER             RECLAIMPOLICY   VOLUMEBINDINGMODE      ALLOWVOLUMEEXPANSION   AGE
+storageclass.storage.k8s.io/local-path   rancher.io/local-path   Delete          WaitForFirstConsumer   false                  3h10m
+```
+
+## Step 3 - Create the StorageOS namespace
+
+```bash
+kubectl create namespace storageos
+```
+
+## Step 4 - Customise and install the helm chart
+
+We make a few changes to the default helm chart values, so we can run in smaller non-production sized clusters.
+
+> ⚠️ Make sure to set the values of `ONDAT_USERNAME` and `ONDAT_PASSWORD`
```

Comment:
If it's a testing environment it doesn't matter, even if it's either "admin"/"admin" or "storageos"/"storageos"

If you keep both `export ONDAT_...` then this line can be removed and _I think_ anyone will understand it can be changed